src/compiler/utils/push_array.ts

```diff
@@ -0,0 +1,11 @@
+export function push_array(thisArray: any[], ...otherList: any[]) {
```

Comment:
actually, do we even need this method? It looks like we could just call `concat` everywhere this is being called for the same effect

Reply:
\*usually\* push is faster than concat. depends on the exact input data. i can add the concat version for benchmarks

Reply:
benchmarks to the rescue ... in the worst case, this is hot code